basyx/aas/adapter/http.py

```diff
@@ -343,8 +343,7 @@ def json_list(cls, data: Union[str, bytes], expect_type: Type[T], stripped: bool
         return [cls.assert_type(obj, expect_type) for obj in parsed]
 
     @classmethod
-    def base64urljson_list(cls, data: str, expect_type: Type[T], stripped: bool, expect_single: bool)\
-            -> List[T]:
+    def base64urljson_list(cls, data: str, expect_type: Type[T], stripped: bool, expect_single: bool) -> List[T]:
```

Comment:
Maybe we can already adapt the line length of 72 in this file (see #285)?

Reply:
Answered in #285